src/wallet/rpcdump.cpp

```diff
@@ -1546,7 +1546,7 @@ static UniValue ProcessDescriptorImport(CWallet& wallet, const UniValue& data, c
         auto existing_spk_manager = wallet.GetDescriptorScriptPubKeyMan(w_desc);
         if (existing_spk_manager) {
             if (!existing_spk_manager->CanUpdateToWalletDescriptor(w_desc, error)) {
-                throw JSONRPCError(RPC_INVALID_PARAMS, error);
+                throw JSONRPCError(RPC_INVALID_PARAMETER, error);
```

Comment:
This could just be fixed in the previous commit where this line is added

Reply:
Done